docs/src/main/sphinx/connector/lakehouse.md

```diff
@@ -0,0 +1,94 @@
+# Lakehouse connector
+
+The Lakehouse connector combines the features of the
+[Hive](/connector/hive), [Iceberg](/connector/iceberg),
+[Delta Lake](/connector/delta-lake), and [Hudi](/connector/hudi)
+connectors into a single connector. It allows you to query or write
+to data stored in multiple table types (also known as table formats)
+that all share the same file system and metastore service.
+
+## General configuration
+
+To configure the Lakehouse connector, create a catalog properties file
+`etc/catalog/example.properties` with the following content, replacing the
+properties as appropriate:
+
+```text
+connector.name=lakehouse
+```
+
+You must configure a [metastore for metadata](/object-storage/metastores).
+The `hive.metastore` property will also configure the Iceberg catalog.
+Do not specify `iceberg.catalog.type`.
+
+You must select and configure one of the
+[supported file systems](lakehouse-file-system-configuration).
+
+## Configuration properties
+
+The following configuration properties are available:
+
+:::{list-table}
+:widths: 30, 58, 12
+:header-rows: 1
+
+* - Property name
+  - Description
+  - Default
+* - `lakehouse.table-type`
+  - The default table type for newly created tables when the `format`
+    table property is not specified. Possible values:
+    * `HIVE`
+    * `ICEBERG`
+    * `DELTA`
+  - `ICEBERG`
+:::
+
+(lakehouse-file-system-configuration)=
+## File system access configuration
+
+The connector supports accessing the following file systems:
+
+* [](/object-storage/file-system-azure)
+* [](/object-storage/file-system-gcs)
+* [](/object-storage/file-system-s3)
+* [](/object-storage/file-system-hdfs)
```

Comment:
add local as well

Reply:
We don't have that for other connectors. Should we?